syntax = "proto3";

import "PB_Menu.proto";
import "PB_ReqHello.proto";
import "PB_ReqPing.proto";
import "PB_ReqPong.proto";

message PB_ExampleOutputMessage {
    oneof type {
        PB_Menu menu = 1;
        bool reset = 2;
        PB_ReqHello req_hello = 3;
        PB_ReqPing req_ping = 4;
        PB_ReqPong req_pong = 5;
    }
}
